tests/unittests/tests-mtd/tests-mtd.c

```diff
@@ -43,85 +46,9 @@
 #define WRITE_SIZE 1
 #endif
 
-static uint8_t dummy_memory[PAGE_PER_SECTOR * PAGE_SIZE * SECTOR_COUNT];
-
-static int _init(mtd_dev_t *dev)
-{
-    (void)dev;
-
-    memset(dummy_memory, 0xff, sizeof(dummy_memory));
-    return 0;
-}
-
-static int _read(mtd_dev_t *dev, void *buff, uint32_t addr, uint32_t size)
-{
-    (void)dev;
-
-    if (addr + size > sizeof(dummy_memory)) {
-        return -EOVERFLOW;
-    }
-    memcpy(buff, dummy_memory + addr, size);
-
-    return 0;
-}
-
-static int _write(mtd_dev_t *dev, const void *buff, uint32_t addr, uint32_t size)
-{
-    (void)dev;
-
-    if (addr + size > sizeof(dummy_memory)) {
-        return -EOVERFLOW;
-    }
-    if (((addr % PAGE_SIZE) + size) > PAGE_SIZE) {
-        return -EOVERFLOW;
-    }
-    memcpy(dummy_memory + addr, buff, size);
-
-    return 0;
-}
-
-static int _erase(mtd_dev_t *dev, uint32_t addr, uint32_t size)
-{
-    (void)dev;
-
-    if (size % (PAGE_PER_SECTOR * PAGE_SIZE) != 0) {
-        return -EOVERFLOW;
-    }
-    if (addr % (PAGE_PER_SECTOR * PAGE_SIZE) != 0) {
-        return -EOVERFLOW;
-    }
-    if (addr + size > sizeof(dummy_memory)) {
-        return -EOVERFLOW;
-    }
-    memset(dummy_memory + addr, 0xff, size);
-
-    return 0;
-}
-
-static int _power(mtd_dev_t *dev, enum mtd_power_state power)
-{
-    (void)dev;
-    (void)power;
-    return 0;
-}
+MTD_EMULATED_DEV(0, SECTOR_COUNT, PAGE_PER_SECTOR, PAGE_SIZE, WRITE_SIZE);
```

Comment:
```suggestion
MTD_EMULATED_DEV(0, SECTOR_COUNT, PAGE_PER_SECTOR, PAGE_SIZE);
```

So CI will be happy :)

Reply:
Oh, I forgot to remove an intermediate change that I had introduced especially for the unittest, but then didn't use after all. Fixed.